src/pages/index.vue

```diff
@@ -35,19 +40,124 @@ const WOW = !isServer ? require('wow.js') : null
 
 import WhatWeDoing from '../components/home/what-we-do'
 import AboutUs from '../components/home/about-us'
+import BlogList from '../components/blogs/blog-list'
 
 export default {
   components: {
     WhatWeDoing,
-    AboutUs
+    AboutUs,
+    BlogList
   },
   data() {
     return {
       currentSection: null,
       sectionList: [
         '#introduction',
         '#about-us',
-        '#what-we-do'
+        '#what-we-do',
+        '#blogs'
+      ],
+      listBlog: [
```

Comment:
listblogs trang home chưa xử lý get 4 cái đầu tiên

Reply:
rồi mà a 
![image](https://user-images.githubusercontent.com/38000690/80934249-7d79f200-8df1-11ea-9e7e-4e8a891de09a.png)

Reply:
e for qua thèn blogData à